lib/Web/API.hs

```diff
@@ -55,27 +54,27 @@ type API
       :> RequiredParam "old-path" (QualifiedPath Path)
       :> RequiredParam "new-path" (QualifiedPath Path)
       :> QueryFlag     "force"
-      :> Post '[JSON] RenameResult
+      :> Post '[JSON] [(EntryPath, EntryPath)]
 
     :<|> "copy"
       :> QueryFlag     "dry-run"
       :> RequiredParam "old-path" (QualifiedPath Path)
       :> RequiredParam "new-path" (QualifiedPath Path)
       :> QueryFlag     "force"
-      :> Post '[JSON] CopyResult
+      :> Post '[JSON] [(EntryPath, EntryPath)]
 
     :<|> "delete"
       :> QueryFlag     "dry-run"
       :> RequiredParam "path" (QualifiedPath Path)
       :> QueryFlag     "recursive"
-      :> Delete '[JSON] DeleteResult
+      :> Delete '[JSON] Directory
```

Comment:
`DELETE` endpoints usually return a 200 with an empty response body when successful.

```suggestion
      :> DeleteNoContent
```

(I guess `/delete-field` is a bit different because it doesn't delete _the whole_ resource, just a part of it, so I guess in that particular case it makes sense to return the `Entry`)

Reply:
Done